content/hi/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Linux पर kubectl इंस्टॉल और सेट करें
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Linux पर kubectl इंस्टॉल करें
+---
+
+## {{% heading "prerequisites" %}}
+
+आपको kubectl संस्करण का उपयोग करना चाहिए जो आपके क्लस्टर के एक मामूली संस्करण अंतर के भीतर हो. उदाहरण के लिए,  v{{< skew latestVersion >}} क्लाइंट निम्नलिखित के साथ संवाद कर सकते हैं v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, and v{{< skew nextMinorVersion >}} कण्ट्रोल प्लेन.
+kubectl के नवीनतम संस्करण का उपयोग करने से अप्रत्याशित मुद्दों से बचने में मदद मिलती है.
```

Comment:
The source page has changed since you drafted this, I think. If you have time, it'd be great to check and to update if this is now stale.

Reply:
cc @vedant-kakde